.circleci/config.yml

```diff
@@ -34,6 +34,37 @@ jobs:
             - /home/circleci/.cache/Cypress
           key: v1-visual{{checksum "package.json"}}
       - run: sh ./scripts/circleci/visual.sh
+  website:
+    docker:
+      - image: circleci/node:dubnium
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-website{{checksum "package.json"}}
+      - run: npm install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-website{{checksum "package.json"}}
+      - run: npm run website:render
+      - run: npm run website:deploy
+  dev:
+    docker:
+      - image: circleci/node:dubnium
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dev{{checksum "package.json"}}
+      - run: npm install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dev{{checksum "package.json"}}
+      - run: npm run angular:dev:build
+      - run: npm run angular:dev:deploy
+
```

Comment:
Is it faster to run the dev app and website deploy as separate jobs in parallel or append to the existing build job and deploy from there? Since the main job has already built the libs and dev apps it would just need to then build the website and deploy.

Reply:
I thought about it quickly but wanted to keep it separate for now since this is somewhat urgent to get around the build limit. 

We could adopt this now and then talk about how to organize the jobs most efficiently?

Reply:
Sounds good!